frontend/src/App.tsx

```diff
@@ -36,7 +42,7 @@ const App = () => {
     toggle();
 
     if (item.id) {
-      const response = await axios.put(`/api/todos/${item.id}/`, item)
+      const response = await axios.put(`/api/todos/${item.id}/`, item, { headers: { 'X-CSRFToken': csrftoken } })
```

Comment:
no change required: if this were real, you'd want to use something like https://axios-http.com/docs/config_defaults to automatically handle csrf token s